aten/src/ATen/native/PadNd.cpp

```diff
@@ -178,7 +178,8 @@ Tensor _pad_circular_symint(const Tensor &self, c10::SymIntArrayRef padding) {
 Tensor _pad_enum_symint(const Tensor &self, c10::SymIntArrayRef pad, int64_t mode_int, c10::optional<double> value) {
   const auto input_dim = self.dim();
   TORCH_CHECK(pad.size() % 2 == 0, "Padding length must be divisible by 2");
-  TORCH_CHECK(static_cast<int64_t>(pad.size()) <= input_dim * 2, "Padding length too large");
+  TORCH_CHECK(static_cast<int64_t>(pad.size()) <= input_dim * 2,
+              "Input dimension should be at least ", pad.size() / 2, " but got ", input_dim);
```

Comment:
The following might be clearer
```suggestion
  TORCH_CHECK(static_cast<int64_t>(pad.size()) <= input_dim * 2,
              "Padding length should be less than or equal to two times the input dimension but got padding length ", pad.size(), " and input of dimension ", input_dim);
```